### ISSUE TITLE
Show cmd line equivalent

### ISSUE THREAD

**Issue by jakearchibald** [jakearchibald/svgomg#11]
I'm not interested in a batch mode, but a view that shows the equivalent cmd will enable this

**Comment by Dogbombs** [comment]
Absolutely.  Anyone with a reasonable amount of command-line Fu should be able to take the command shown and put it in a batch.  Thanks!

**Comment by nobodypb** [comment]
+1
Also a svgo settings file (.svgo.yml) with the current settings for download would be nice

**Comment by Tigt** [comment]
Might help when debugging as well, for when those rogue SVGs that SVGO chokes on get input.

**Comment by bkuri** [comment]
+1
PS, thanks for the awesome tool.

**Comment by colinmcglynn** [comment]
+1
I would love to be able to use this tool to test out some different settings and then use the exported settings to run a batch on the command line.

**Comment by motleydev** [comment]
+1

**Comment by harold** [comment]
+1

**Comment by WebMechanic** [comment]
There are now 3 separate request **since 2015** to implement this feature in one way or another: this one #11, #50 and #184 (closed in favour of this).
